app/routes/reminders.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
```

Comment:
Mirage is generating your models behind the scenes. Look into how to grab records from the store instead of hard coding them in here.